qualification/report/generate_pdf.py

```diff
@@ -835,14 +826,64 @@ def document_from_json(input_data: str | list) -> Document:
     return doc
 
 
+def test_image_commit(result_list: list) -> str:
+    """Extract the git commit ID of the test image from the test results.
+
+    Parameters
+    ----------
+    result_list: list
+        A list containing the test results.
+
+    Returns
+    -------
+    commit_id: str
```

Comment:
```suggestion
    commit_id : str
```
Based on the [style guide](https://numpydoc.readthedocs.io/en/latest/format.html)